src/components/TimeAndSalary/common/formatter.js

```diff
@@ -110,3 +110,7 @@ export const formatWage = wage => {
 
 export const formatDate = ({ year, month }) =>
   `${year}.${month >= 10 ? '' : 0}${month}`;
+
+export const getReportCount = (salaryWorkTimeReports = 0) => {
```

Comment:
從 callsite 來看 input 應該是 Array？
```suggestion
export const getReportCount = (salaryWorkTimeReports = []) => {
```